examples/dnn_dcgan_train_ex.cpp

```diff
@@ -134,8 +134,8 @@ int main(int argc, char** argv) try
     // setup all leaky_relu_ layers in the discriminator to have alpha = 0.2
     visit_computational_layers(discriminator, [](leaky_relu_& l){ l = leaky_relu_(0.2); });
     // Remove the bias learning from all bn_ inputs in both networks
-    set_all_bn_inputs_no_bias(generator);
-    set_all_bn_inputs_no_bias(discriminator);
+    disable_duplicative_bias(generator);
+    disable_duplicative_bias(discriminator);
```

Comment:
@davisking, I noticed that we agreed to call it `disable_duplicative_biases`, but somehow I just called like this... Should we rename it before the stable release? I think it should be plural, too.

Reply:
Huh, yeah.  `disable_duplicative_biases` is probably a better name.  If you want to do that rename in a separate PR that would be cool.

Reply:
Sorry about that...

Reply:
No worries :)